roomserver/internal/perform/perform_join.go

```diff
@@ -314,7 +314,7 @@ func (r *Joiner) performJoinRoomByID(
 		// sign the event with the pseudo ID key
 		identity = fclient.SigningIdentity{
 			ServerName: "self",
-			KeyID:      "ed25519",
+			KeyID:      "ed25519:1",
```

Comment:
what does the 1 mean?

Reply:
We have a `isValidID` in GMSL, which checks for a `:`, so added this. It's basically unused.